pymysqlreplication/gtid.py

```diff
@@ -204,6 +210,14 @@ def encoded_length(self):
                 len(self.intervals))
```

Comment:
I am wondering what is this `2 *  # stop/start` in binary encoding and why only `8` for stop/start.

- [x] SID ok
- [x] interval len
- [ ]  2 *  # stop/start ??
- [ ] stop / start as 2 u64 OK


later we do for `start/stop`  reading: `start, end = struct.unpack('<QQ', payload.read(16))`

and writting is

```
        buffer += struct.pack('<Q', len(self.intervals))
        for interval in self.intervals:
            # Start position
            buffer += struct.pack('<Q', interval[0])
            # Stop position
            buffer += struct.pack('<Q', interval[1])
```

I am struggling with this 2 and 8. Any clue? If it's a mistery I can write a test XD

I have not seen explicit testing of `Gtid.encoded_length`.